src/DOM/HTML/Indexed.purs

```diff
@@ -328,6 +329,7 @@ type HTMLinput = Interactive
   ( accept :: MediaType
   , autocomplete :: OnOff
   , autofocus :: Boolean
+  , capture :: Boolean
```

Comment:
AFAICS that's a draft and the published spec says it's a boolean:

https://www.w3.org/TR/html-media-capture/#the-capture-attribute